Framework/Algorithms/src/PDFFourierTransform.cpp

```diff
@@ -296,12 +358,14 @@ void PDFFourierTransform::exec() {
     double fs = 0;
     double error = 0;
     for (size_t q_index = qmin_index; q_index < qmax_index; q_index++) {
-      double q = inputQ[q_index];
-      double deltaq = inputQ[q_index] - inputQ[q_index - 1];
+      const double q = inputQ[q_index];
+      const double deltaq = inputQ[q_index] - inputQ[q_index - 1];
       double sinus = sin(q * r) * deltaq;
+
       // multiply by filter function sin(q*pi/qmax)/(q*pi/qmax)
       if (filter && q != 0) {
-        sinus *= sin(q * rdelta) / (q * rdelta);
+        const double lorchKernel = q * M_PI / inputQ[qmax_index];
+        sinus *= sin(lorchKernel) / lorchKernel;
```

Comment:
That factor was hiding in the default value of `rdelta`. This version gets rid of the assumption that the user didn't override `rdelta`.